src/main/pages/che-7/installation-guide/proc_che-usage-with-tls.adoc

```diff
@@ -0,0 +1,46 @@
+[id="{prod-id-short}-usage-with-tls_{context}"]
+= {prod-short} usage with TLS
+
```

Comment:
This is probably should be marked as Linux only. For OSX it looks like that https://support.securly.com/hc/en-us/articles/206058318-How-to-install-the-Securly-SSL-certificate-on-Mac-OSX-
TODO need to check on Windows.

Reply:
It's more "browser only" and should be imho independent on OS. Your link shows how to install system-wide certificate, which then should be recognized by the browser. However, it still should be possible to install certificate to browser only on OSX. Please correct me if I'm wrong.

I agree we should mention this. I'll send a patch.

Reply:
@mvala can you add note about this please ?

Reply:
I have some doubts that there is such a thing "browser only" for OSX+chrome
![output](https://user-images.githubusercontent.com/1614429/71818069-2c96fe80-3088-11ea-8c61-0d64a3e0356e.gif)

Reply:
aha, didn't know that. I've added it https://github.com/eclipse/che-docs/pull/1007/files#diff-d019fdb3755c69680e4c29aee7e2c362R50 I guess test it and screenshots are on you (if you want)